mne/viz/_3d.py

```diff
@@ -666,8 +856,8 @@ def plot_trans(info, trans='auto', subject=None, subjects_dir=None,
 
     for key in surfs.keys():
         surfs[key] = transform_surface_to(surfs[key], coord_frame, mri_trans)
-    src_rr = apply_trans(mri_trans, src_rr)
-    src_nn = apply_trans(mri_trans, src_nn, move=False)
+    # src_rr = apply_trans(mri_trans, src_rr)
+    # src_nn = apply_trans(mri_trans, src_nn, move=False)
```

Comment:
I think this was not working correctly. See https://1709-35097627-gh.circle-artifacts.com/0/tmp/circle-artifacts.Ix5gkZV/html/auto_tutorials/plot_surface_alignment.html (built before this change).